sklearn/metrics/tests/test_classification.py

```diff
@@ -2738,7 +2738,22 @@ def test_log_loss_pandas_input():
         assert_allclose(loss, 0.7469410)
 
 
-def test_brier_score_loss():
+def test_log_loss_warnings():
+    expected_message = re.escape(
+        "Labels passed were ['spam', 'eggs', 'ham']. But this function "
+        "assumes labels are ordered lexicographically. "
+        "Ensure that labels in y_prob are ordered as "
+        "['eggs' 'ham' 'spam']."
```

Comment:
```suggestion
        "Labels passed were ['spam', 'eggs', 'ham']. But this function assumes "
        "labels are ordered lexicographically. Ensure that labels in y_prob are "
        "ordered as ['eggs' 'ham' 'spam']."
```

Reply:
"Ensure that labels in y_prob are ordered as ..." could be rephrased to be easier to grasp. For instance: "Ensure that columns of y_prob match the following label ordering: ...".